lib/ext/tags.js

```diff
@@ -0,0 +1,96 @@
+'use strict';
```

Comment:
This actually isn't a moved file, it's the file built from `src/ext/tags.js`.  The source file itself, `src/ext/tags.js` is unchanged by this PR.

All the files in `lib` are the down-compiled ES5 JavaScript built from the ES6 source. To the best of my knowledge, it's a standard convention to check-in these built files so consumers of the library do not need to install and run babel.